src/Ninja/G1CP/Content_G1.src

```diff
@@ -39,6 +39,7 @@ Content\Misc\vobLight.d
 Content\Misc\npc.d
 Content\Misc\ou.d
 Content\Misc\routine.d
+Content\Misc\constant.d
```

Comment:
I think this function should not be in this collection of files. As it is rather low level (replacing a string in an arbitrary string array), it should be added to the file `src/Ninja/G1CP/Content/Symbols/string.d` where the other functions used reside.

Reply:
Agreed.